### DISCUSSION TITLE
Daughter board for Raspberry Pi ?

### DISCUSSION THREAD

**Discussion by trudK45** [peterantypas/maiana#40] [Ideas]
Wouldn't it be nice to make the logic board fit on to a raspberry pi ?
Something like this : [https://www.rooco.eu/products/moitessier-hat-2-raspberry-pi-openplotter-ais-receiver/](url)
Difference being you will have a transponder.
This shouldn't be to much of a job, should it ?

**Comment by peterantypas** [comment]
I will not do this. MAIANA&trade; is an antenna with a built-in transponder and GNSS. Absolutely no analog signals to and from it. That's the main differentiator.

**Comment by peterantypas** [comment]
That said, the control circuit itself *could* be a Pi Hat, but I don't plan to work on that -- no bandwidth.
